packages/z/zonos-elements.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "zonos-elements",
+  "filename": "dist/scripts/loadZonos.js",
+  "description": "Zonos component UI for Stripe elements",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
+  "unpkg": "dist/zonos-elements/zonos-elements.esm.js",
+  "collection": "dist/collection/collection-manifest.json",
+  "keywords": [
+    "cross-border",
+    "international shopping",
+    "calculate-landed-cost",
+    "zonos",
+    "elements",
+    "payments",
+    "checkout"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Zonos/zonos-elements-readme"
+  },
+  "files": [
+    "dist/",
+    "loader/"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@zonos/elements",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*"
+        ]
```

```suggestion
        "basePath": "dist/zonos-elements",
        "files": [
          "*.@(css|js)"
        ]
```
Please avoid using a globstar as it can lead to including files that are not meant to be on the CDN. I suspect you want to include specifically the JS + CSS files under `dist/zonos-elements`. (And given that, I suspect you'll want the `filename` property in this JSON file, which defines the default file recommended, to be `zonos-elements.esm.js`).

---

Thank you for your feedback. My default file for the CDN to load is `dist/scripts/loadZonos.js`, which I have specified in the `filename` at the root. This will then load all the necessary files from the `dist/zonos-elements` folder as required. I will make the adjustments and ask for your review.